src/AppInstallerCLICore/Argument.cpp

```diff
@@ -97,6 +97,10 @@ namespace AppInstaller::CLI
             return Argument{ "wait", NoAlias, Args::Type::Wait, Resource::String::WaitArgumentDescription, ArgumentType::Flag, false };
         case Args::Type::ProductCode:
             return Argument{ "product-code", NoAlias, Args::Type::ProductCode, Resource::String::ProductCodeArgumentDescription, ArgumentType::Standard, false };
+        case Args::Type::DisplayNotes:
+            return Argument{ "display-notes", NoAlias, Args::Type::DisplayNotes, Resource::String::DisplayNotesArgumentDescription, ArgumentType::Flag, false };

```

Comment:
Could we ever want to add another type of notes that may make this ambiguous? Say, if we wanted to show both installation notes and change notes.

Reply:
I would imagine any other type of notes that we want to optionally display could be grouped together and controlled by these flags.

Reply:
Do we even really need a command line argument for this?  I don't see it being desired enough to warrant adding it.  Maybe if someone complains we could do so later, but for now I would not.

Reply:
Removed added command line arguments